ci-operator/config/openshift/azure-storage-azcopy/openshift-azure-storage-azcopy-main.yaml

```diff
@@ -0,0 +1,42 @@
+base_images:
+  ocp_4.16_base:
+    name: "4.16"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
```

Comment:
Go version should be defined in openshift/azure-storage-azcopy repository in .ci-operator.yaml, so ART can change it easily. Here we need:

```
build_root:
  from_repository: true
```

In addition, I am not sure we support 1.19 go images in 4.16.

Reply:
@jsafrane I see, that's better. I followed what we have in efs-utils and we don't use `from_repository` there. I'm adding ci-operator config to azcopy repo, should be merged first: https://github.com/openshift/azure-storage-azcopy/pull/3

Reply:
efs-utils are in python, I'm not sure why we use a go base image there... Maybe because it just works.